### DISCUSSION TITLE
Provide a way to automatically add multiple discount codes in checkout (for non-Shopify Plus)

### DISCUSSION THREAD

**Discussion by cyberabis** [Shopify/function-examples#224] [Feedback: Discounts APIs]
We see that for Shopify Plus there is a [Checkout UI Extension property](https://shopify.dev/docs/api/checkout-ui-extensions/apis/standardapi#properties-propertydetail-applydiscountcodechange) that allows an app to automatically add up to 5 discount codes in the checkout.

Can we have this for non-Shopify Plus stores too? Or, otherwise if we can enhance the "discount permalink - https://<your_shop_url>/discount/<your_discount_code>" to accept multiple comma-separated discount codes, that will also help.

**Why we need this?**
Currently apps are limited to creating maximum 5 automatic discounts which is quite low for apps like volume discounts.
One way to overcome this limitation is to use discount codes, but apps need a way to tell the checkout on what discount codes need to be applied.

**Related Slack discussion:**
https://shopifypartners.slack.com/archives/C0225DTE1M1/p1681827089161589

**Comment by irahulranjan** [comment]
@cyberabis found any workaround for this?

**Reply by cyberabis** [reply]
Nope @irahulranjan, Maybe share your use case in this thread and upvote as well. We can see if the Shopify team picks this up!

**Comment by cyberabis** [comment]
This can be closed now as there appears to be a way now just as mentioned in the problem.

**Reply by irahulranjan** [reply]
can you share a sample permalink with multiple codes?
